app/src/main/java/fr/free/nrw/commons/upload/UploadMediaDetailAdapter.java

```diff
@@ -203,51 +205,53 @@ public void removeDescription(final UploadMediaDetail uploadMediaDetail, final i
 
     public class ViewHolder extends RecyclerView.ViewHolder {
 
-        @Nullable
-        @BindView(R.id.description_languages)
-        TextView descriptionLanguages;
+        TextView descriptionLanguages ;
 
-        @BindView(R.id.description_item_edit_text)
         PasteSensitiveTextInputEditText descItemEditText;
 
-        @BindView(R.id.description_item_edit_text_input_layout)
         TextInputLayout descInputLayout;
 
-        @BindView(R.id.caption_item_edit_text)
         PasteSensitiveTextInputEditText captionItemEditText;
 
-        @BindView(R.id.caption_item_edit_text_input_layout)
         TextInputLayout captionInputLayout;
 
-        @BindView(R.id.btn_remove)
         ImageView removeButton;
 
-        @BindView(R.id.btn_add)
         ImageView addButton;
 
-        @BindView(R.id.cl_parent)
         ConstraintLayout clParent;
 
-        @BindView(R.id.ll_write_better_caption)
         LinearLayout betterCaptionLinearLayout;
 
-        @BindView(R.id.ll_write_better_description)
         LinearLayout betterDescriptionLinearLayout;
```

Comment:
`TextInputeLayout, PasteSensitiveTextInputEditText, TextInputLayout, ImageView, ConstraintLayout, LinearLayout `

don't define variable for this its no longer needed in view binding it was needed in butterknife as we needed annotation. 
Remove this and its imports as well and optimize import.

Reply:
Again
<img width="618" alt="Screenshot 2024-03-09 at 1 44 05 PM" src="https://github.com/commons-app/apps-android-commons/assets/126143257/fe1445c8-4575-4524-9fbe-a9a9aa92f7ec">